src/test/java/nextstep/fp/CarTest.java

```diff
@@ -8,24 +8,14 @@
     @Test
     public void 이동() {
         Car car = new Car("pobi", 0);
-        Car actual = car.move(new MoveStrategy() {
-            @Override
-            public boolean isMovable() {
-                return true;
-            }
-        });
+        Car actual = car.move(()->true);
```

Comment:
```suggestion
        Car actual = car.move(() -> true);
```

코드 스타일이 맞지 않네요. 인텔리제이에서 제공하는 오토 포매팅 기능을 이용해보는건 어떨까요 ?